pkg/cosign/tuf/client_test.go

```diff
@@ -127,10 +128,14 @@ func TestValidMetadata(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unexpected error")
 	}
+	if cl, isCloser := local.(io.Closer); isCloser {
+		// TODO: this is a hack to free the file descriptors, need to patch `tuf_leveldbstore.FileLocalStore` to return a io.Closer
```

Comment:
Open an issue to track?

Reply:
https://github.com/sigstore/cosign/issues/1160

Reply:
also added link in a commit